scripts/xe-reset-networking

```diff
@@ -184,6 +187,8 @@ Type 'no' to cancel.
 		f = file(management_conf, 'w')
```

Comment:
As we have discussed in person: we'll keep the commit here but create a ticket to improve the Pyhton code to use a better templating mechanism.